xmipptomo/protocols/protocol_detect_misalignment_TS.py

```diff
@@ -0,0 +1,690 @@
+# **************************************************************************
+# *
+# * Authors:     Federico P. de Isidro Gomez (fp.deisidro@cnb.csi.es) [1]
+# *
+# * [1] Centro Nacional de Biotecnologia, CSIC, Spain
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 2 of the License, or
+# * (at your option) any later version.
+# *
+# * This program is distributed in the hope that it will be useful,
+# * but WITHOUT ANY WARRANTY; without even the implied warranty of
+# * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# * GNU General Public License for more details.
+# *
+# * You should have received a copy of the GNU General Public License
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307  USA
+# *
+# *  All comments concerning this program package may be sent to the
+# *  e-mail address 'scipion@cnb.csic.es'
+# *
+# **************************************************************************
+
+import os
+
+from pwem.emlib import lib
+import pwem.emlib.metadata as md
+from pyworkflow import BETA
+import pyworkflow.protocol.params as params
+import pyworkflow.utils.path as path
+from pyworkflow.object import Set, List, String, Boolean
+from pyworkflow.protocol.constants import STEPS_PARALLEL
+from pwem.protocols import EMProtocol
+import tomo.objects as tomoObj
+from tomo.protocols import ProtTomoBase
+import xmipptomo.utils as utils
+import emtable
+
+METADATA_INPUT_COORDINATES = "fiducialCoordinates.xmd"
+VRESMOD_FILE_NAME_EXT = "_vResMod.xmd"
+
+
+class XmippProtDetectMisalignmentTiltSeries(EMProtocol, ProtTomoBase):
+    """
+    Scipion protocol for xmipp_tomo_detect_misalignment_trajectory. Detect misalignment in a tilt series.
```

Comment:
A help is needed to find the protocol in scipion

Reply:
Done